roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -9,6 +9,9 @@ class PackageVersion(NotContainerizedMixin, OpenShiftCheck):
     name = "package_version"
     tags = ["preflight"]
 
+    recommended_ovs_version_os_3_5 = "2.6"
+    recommended_ovs_version = "2.4"
```

Comment:
This seems hard to evolve when newer releases are out.

Which makes me think how would we proceed once 3.6 is out. Are we expecting to be revisiting the checks every now and then and updating the table of version numbers?